packages/spark-core/base/inputs/_error-container.scss

```diff
@@ -0,0 +1,17 @@
+.sprk-b-ErrorContainer {
+  align-items: center;
+  display: flex;
+  margin-left: $space-s;
+  margin-top: $space-s;
+}
+
+.sprk-b-ErrorIcon {
+  fill: $text-input-error-color;
+  margin-right: $space-s;
+}
+
+.sprk-b-ErrorText {
+  color: $black;
```

Comment:
Does Error text need this font color declaration since this will be the default type color?

Reply:
ill move it out to a var